src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+[class^="stars--"] {
```

Comment:
The selector should use `stars` class as the base class and then use the extra classes `stars--0`, `stars--1`, `stars--2`, ... `stars--5`. Use `.stars.stars--0`, `.stars.stars--1`, and so on instead of [class^='stars--']